jupyterlab_bigquery/jupyterlab_bigquery/details_handler/details_handler.py

```diff
@@ -14,6 +14,8 @@
 from collections import namedtuple
 from notebook.base.handlers import APIHandler, app_log
 from google.cloud import bigquery
+from google.cloud.bigquery.enums import StandardSqlDataTypes, SqlTypeNames
+from google.cloud.bigquery_v2.gapic.enums import Model
```

Comment:
`Model` is imported for model types, while `StandardSqlDataTypes` is an enum for sql types and `SqlTypeNames` converts sql types to display names.